### ISSUE TITLE
Update for latest DLC patch 

### ISSUE THREAD

**Issue by orchidalloy** [JohnEdwa/RTAutoSprintExtended#16]
It doesn't work right now

**Comment by Araeos** [comment]
Same here. For me this mod currently does not automatically keep sprinting active at all. It used to work for me before the dlc though.

**Comment by Araeos** [comment]
It seems that installing HookGenPatcher by RiskofThunder seems to enable RTAutoSprintEx to work. I tried it since the author pushed a commit with the unreleased update 2.1.1 to fix this it is just a matter of updating the mod release from the current 2.1.0 to include this dependency for people who don't have the HookGenPatcher already insalled.

**Comment by Kommynct** [comment]
It works mostly for me now, but it doesn't let me change the sprint fov anymore, it just uses the default no matter what it's set to

**Comment by JohnEdwa** [comment]
Yup, a known issue. The DLC completely changed how the camera system works and I wasn't able to find where the sprint FOV change is being done anymore.

**Comment by Kommynct** [comment]
https://github.com/dma14/RoR2_AddFoVSettings this person seems to have found a way to do it.

**Comment by RiceIRL** [comment]
Needs another fix for the new SoTS DLC update, it seems.
